src/static/css/skip-nav.less

```diff
@@ -0,0 +1,24 @@
+#skip-nav {
+    padding: 10px;
```

Comment:
Crud, I was thinking border was in Alpha, but we're not using `box-sizing: border` are we?